code/scipy/signal/signal.c

```diff
@@ -120,11 +122,180 @@ mp_obj_t signal_sosfilt(size_t n_args, const mp_obj_t *pos_args, mp_map_t *kw_ar
 MP_DEFINE_CONST_FUN_OBJ_KW(signal_sosfilt_obj, 2, signal_sosfilt);
 #endif /* ULAB_SCIPY_SIGNAL_HAS_SOSFILT */
 
+#if ULAB_SCIPY_SIGNAL_HAS_OACONVOLVE
+
+mp_obj_t signal_oaconvolve(size_t n_args, const mp_obj_t *pos_args, mp_map_t *kw_args)
+{
```

Comment:
This doesn't conform to `micropython`'s coding standards: https://github.com/micropython/micropython/blob/master/CODECONVENTIONS.md#c-code-conventions
`Put opening braces on the end of the line it belongs to, not on a new line.`